src/java/org/apache/cassandra/db/compaction/CompactionManager.java

```diff
@@ -232,26 +234,40 @@ public static void storeControllerConfig()
     @VisibleForTesting
     public static void cleanupControllerConfig()
     {
-        Pattern fileNamePattern = Pattern.compile("-controller-config.JSON", Pattern.LITERAL);
+        Pattern fileNamePattern = Pattern.compile(CONTROLLER_CONFIG_JSON_SUFFIX, Pattern.LITERAL);
         Pattern keyspaceNameSeparator = Pattern.compile("\\.");
         File dir = DatabaseDescriptor.getMetadataDirectory();
         if (dir != null)
         {
             for (File file : dir.tryList())
             {
-                if (file.name().contains("-controller-config.JSON"))
+                if (file.name().contains(CONTROLLER_CONFIG_JSON_SUFFIX))
                 {
                     String[] names = keyspaceNameSeparator.split(fileNamePattern.matcher(file.name()).replaceAll(""));
-                    try
+                    if (names.length == 2)
```

Comment:
nit: can you add some comments that this is related to file name abbreviation in `Controller#getControllerConfigPath`?